### PR TITLE
Update mobile menu design of the portfolio

### PR DESCRIPTION
**Pull request by benjp009**
# 👋 Micronaut Reviewer 

[Portfolio - mobile menu](https://github.com/microverseinc/curriculum-html-css/blob/main/portfolio/7_mobile_menu.md)

### Requirements 🚀

I have reviewed the following on the page for best accessibility : 

For this project: 
- [x] you should add the JavaScript + HTML + CSS linter.

In mobile, implement the following features:

- [x] When the user clicks (or taps) the hamburger button, the mobile menu appears.
- [x] When the user clicks (or taps) the close (X) button, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, the mobile menu disappears.
- [x] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed.
- [x] You must stick to the design as much as possible (e.g., font, colors, images, tests, margins) using the templates in Figma.

### Pair Programming 🎳

For this exercise, PP was required, we worked very closely with @fabianofrank to make the best looking possible interaction of the mobile menu for my portfolio.

Happy Review :)

### CODE REVIEW

File: menu.js [hunk 1]
```diff
@@ -1,4 +1,5 @@
-function clickFunction('') {
+// eslint-disable-next-line no-unused-vars
```
Comment (benjp009) [inline menu.js:1]: We couldn't find out how to fix the linter error `no-unused-vars`, so we added this line of code to pass the linters, if you have any idea why the linter failed, please do let us know, and we will fix it. :)

**Review by de-scholar — CHANGES_REQUESTED** [review]
Hi @benjp009 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -25,12 +27,15 @@
...
-          <a href="#portfolio">Portfolio</a>
-          <a href="#about">About</a>
-          <a href="#contact">Contact</a>
+          <a href="javascript:void(0);" class="icon" onclick="clickFunction()">
+            <i class="fa fa-times" id="close-button" aria-hidden="true"></i>
+          </a>
+          <a href="#portfolio" onclick="clickFunction()">Portfolio</a>
+          <a href="#about" onclick="clickFunction()">About</a>
+          <a href="#contact" onclick="clickFunction()">Contact</a>
         </div>
```
[the remark this answers is not in the document]
Comment (benjp009) [reply index.html:36]: Thanks for your feedbacks @de-scholar 
I will work on the `x` button, actually it's already here on the page, but out of the screen view :) 
--
I will work on that and resubmit

**Comment by benjp009** [comment]
# 👋 Micronaut Reviewer 
## Second Review 

We have done all the changes required by the previous reviewer, including the optional once. I know the design of the page is not perfect, but it fits today requirements about mobile menu JavaScript requirements. 

- [x] The menu is missing the close `X` button to close it
- [ ] [optional] The menu should be full screen, please remove the white space at the very top of the menu bar
- [x] [ optional ]  The full screen version, looks buggy, the menu bar is occupying the whole screen, it would be nice if you implement it

### Thanks for taking the time to review my work :)

**Comment by Davidosky007** [comment]
## Hi @benjp009 👋, 

## Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
## Congratulations! 🎉 



well-done
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by benjp009** [comment]
## Hi @Davidosky007 👋, 

#### Thank you so much for your review and approval !!! 

Thanks
Cheers and Happy coding!👏👏👏

**Pull request merged**